workflow/apm/apm-overview.mdx

```diff
@@ -0,0 +1,23 @@
+---
+title: APM Dashboard - Overview
+sidebarTitle: Overview
+---
+
+The Application Performance Monitoring (APM) dahshboard is a unified dashboard that provides real-time insights into the performance of your applications. Whether instrumenting your APMs with the Infrastructure (Infra) Agent or Serverless, all of your APM data is displayed within this section of the Middleware.
+
+# Infra Agent vs Serverless
+
+There are two different ways to instrument your APMs, through the [Infrastructure Agent]() or a Serverless. The Infra Agent uses OpenTelemetry to collect metrics and traces from your application and send them to Middleware. Instrumenting the APM Agent with the Serverless Agent provides the same features as the Infra Agent apart from the Continuous Profiling feature and slight efficiency degradation. The Infra Agent collects continuous profiling data through Pyroscope, which is not made available when Serverless.
+
+# How APM Data is Collected
+
+The APM uses the OpenTelemetry Collector to gather telemetry data from your application and then send it to Middleware with OpenTelemetry exporters. Telemetry data is collected from the Infrastructure Agents that are instrumenting your APMs to easily offload data to a dedicated exporter.

```

Comment:
Recommend edit: 

"We've built a suite of supported APM packages which automatically track and export data from your services to the Infra Agent, which in turns compresses and batches that data to Middleware (or directly to Middleware if running as Serverless).

Because our packages leverage OpenTelemetry you are also able to customize the data sent through our APMs via any OpenTelemetry supported methods."